Corona-Warn-App/src/main/res/layout/rat_profile_onboarding_fragment.xml

```diff
@@ -15,31 +13,264 @@
\ No newline at end of file
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/rat_profile_onboarding_scroll_view"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginBottom="20dp"
+        app:layout_constraintBottom_toTopOf="@+id/next_button"
+        app:layout_constraintTop_toBottomOf="@id/toolbar">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageView
+                android:id="@+id/rat_profile_onboarding_image"
+                android:layout_width="0dp"
+                android:layout_height="190dp"
+                android:layout_marginHorizontal="24dp"
+                android:layout_marginTop="4dp"
+                android:contentDescription="@string/rat_profile_onboarding_image_content_description"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/rat_profile_onboarding" />
+
+            <TextView
+                android:id="@+id/rat_profile_onboarding_title"
+                style="@style/headline6"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:layout_marginHorizontal="24dp"
+                android:layout_marginTop="24dp"
+                android:text="@string/rat_profile_onboarding_title"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_image" />
+
+            <TextView
+                android:id="@+id/rat_profile_onboarding_subtitle"
+                style="@style/subtitleMedium"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="24dp"
+                android:layout_marginTop="22dp"
+                android:text="@string/rat_profile_onboarding_subtitle"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_title" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/rat_profile_onboarding_card_container"
+                style="@style/cardTracing"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="24dp"
+                android:layout_marginTop="32dp"
+                android:background="#F5F5F5"
+                android:orientation="vertical"
```

Comment:
Does it also have an effect on ConstraintLayouts? Thought it is only for LinearLayouts